primitives/api/src/lib.rs

```diff
@@ -490,12 +490,6 @@ pub trait ApiExt<Block: BlockT> {
 	where
 		Self: Sized;
 
-	/// Returns the version of the given api.
-	fn api_version<A: RuntimeApiInfo + ?Sized>(
```

Comment:
Why we don't need it?

Reply:
It's a dublicate left after merge. Check line 485